### DISCUSSION TITLE
Code editor card

### DISCUSSION THREAD

**Discussion by mturoci** [h2oai/wave#629] [Ideas]
via @srini-x 

#### Is your feature request related to a problem? Please describe

There is currently no card for live-coding apps.

#### Describe the solution you'd like

Provide a card with embedded [monaco-editor](https://microsoft.github.io/monaco-editor/) or other suitable alternative.

#### Questions for discussion
What should a UI look like with best possible UX in mind? This includes:
* how/when would the card be saved
* how it should be marked as dirty

**Comment by lo5** [comment]
Including Monaco by default will bloat up our js bundle. Let's address after #297. In any case, we have this covered in #278.

**Comment by mturoci** [comment]
> Including Monaco by default will bloat up our js bundle

Not necessarily, we can code split it via lazy loading the Monaco js module.

**Comment by srini-x** [comment]
@lo5 much simpler request, can we get an option to have the user entered text in multi-line textbox formatted in monospace font?
This will handle all the cases where user wants to enter a code snippet.
